src/index.js

```diff
@@ -25,6 +25,8 @@ function updateSnapshotState(oldSnapshotState, newSnapshotState) {
 }
 
 function configureToMatchImageSnapshot({
+  customSnapshotIdentifier: commonCustomSnapshotIdentifier = '',
```

Comment:
Why would you want a common snapshot identifier? 2 snapshots sharing the same name would cause only 1 snapshot to ever be written and just keep overriding each other.

Reply:
The idea being that you could use this instead of the directory path. If you had two snapshot names that were otherwise identical without the directory path, you'd re-set this config var before the second screenshot was taken. It lets you group your screenshots by name through some method other than directory. I mostly threw this in there because it was the only config var not currently being "passed through".